### DISCUSSION TITLE
"Starstruck" achievement don't show

### DISCUSSION THREAD

**Discussion by amaargiru** [community/community#40199] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello!

I have repository (https://github.com/amaargiru/pyroad) with 138 stars, but I don't see the "Starstruck" achievement.

"Settings" / "Show Achievements on my profile" is active.

**Comment by radar-blip** [comment]
It may be hidden from view.
Check here for it: https://github.com/amaargiru?tab=achievements
If its listed there, unhide it by clicking the 👁️ icon

**Reply by amaargiru** [reply]
Thanks a lot, your advice helped me.

**Reply by radar-blip** [reply]
You're welcome. Enjoy your new badge!! 😀

**Reply by olyble** [reply]
thank you
